src/engine/input/key_mapping/InputActionItem.cs

```diff
@@ -136,7 +136,7 @@ public override int GetHashCode()
         var inputActionItem = (InputActionItem)target.InputActionItemScene.Instance();
 
         inputActionItem.InputName = data.InputName;
-        inputActionItem.DisplayName = data.Name;
+        inputActionItem.DisplayName = TranslationServer.Translate(data.Name);
```

Comment:
What did this fix?

Reply:
As far as I can tell, the "data.Name" here is already translated like "Move left"

Reply:
somehow, it isn't... I've checked the code, there is a `TranslateFrom` Attribute and the source string exists in the translation file.

Reply:
What text are you seeing and where that isn't translated. When debugging with a breakpoint on this line, it looks like when initially building the options menu, this is not translated, but afterwards it is. Anyway the real fix is to make sure that NamedInputAction gets translated when the game starts.

Reply:
the debugger shows me that Name is always untranslated.

Reply:
Bug seems to be that SimulationParameters.ApplyTranslations doesn't get called when the game loads. It should be called the first time it is loaded. So maybe in SimulationParameters constructor?

Reply:
Actually, I remember that a recent PR by @Kasterisk removed the SimulationParameters.ApplyTranslations call when the parameters are loaded. I suspect that, that should be put back in, at least when the options are initially loaded.

Reply:
yeah, `ApplyTranslations` only gets called when changing the language and not when the game starts

Reply:
I just opened a PR trying to address this problem: https://github.com/Revolutionary-Games/Thrive/pull/2175